### ISSUE TITLE
Accessibility:Panel: State “pressed” / “not pressed” about the ‘Toggle Expand’ button is not read

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3493]
Description: State “pressed” / “not pressed” about the ‘Toggle Expand’ button is not read.
Also, the “Pressed” state is not visually indicated. 

Expected: state ‘pressed’ or ‘not pressed’ should be communicated to the user.
Also, the ‘pressed’ state should be visually indicated

Screenshot: 

![image](https://user-images.githubusercontent.com/32538291/94650708-d902ca00-0314-11eb-9cfc-37c61dab74ed.png)

![image](https://user-images.githubusercontent.com/32538291/94650714-dbfdba80-0314-11eb-85a9-5b6646ed864e.png)
